t/app/controller/waste_4471550.json

```diff
@@ -0,0 +1,188 @@
+                    "ExtensibleDatum": [
+                      { "DatatypeName": "Container Type", "Value": "18" },
+                      { "Value": "1", "DatatypeName": "Quantity" }
+                    ]
+                  },
+                  "DatatypeName": "Recycling Containers"
+                },
+                {
+                  "DatatypeName": "Recycling Containers",
+                  "ChildData": {
+                    "ExtensibleDatum": [
+                      { "DatatypeName": "Container Type", "Value": "21" },
+                      { "Value": "1", "DatatypeName": "Quantity" }
+                    ]
+                  }
+                }
+              ]
+            },
+            "ServiceTaskSchedules": {
+              "ServiceTaskSchedule": {
+                "NextInstance": {
+                  "Ref": { "Value": { "anyType": [ "22252757", "8325" ] } },
+                  "OriginalScheduledDate": { "DateTime": "2022-10-16T23:00:00Z" },
+                  "CurrentScheduledDate": { "DateTime": "2022-10-16T23:00:00Z" }
+                },
+                "EndDate": { "DateTime": "2050-01-01T00:00:00Z" },
+                "StartDate": { "DateTime": "2018-09-02T23:00:00Z" },
+                "LastInstance": {
+                  "Ref": { "Value": { "anyType": [ "22252757", "8311" ] } },
+                  "OriginalScheduledDate": { "DateTime": "2022-10-02T23:00:00Z" },
+                  "CurrentScheduledDate": { "DateTime": "2022-10-03T05:00:00Z" }
+                },
+                "ScheduleDescription": "Monday every other week"
+              }
+            },
+            "Id": "2003",
+            "EndDate": { "DateTime": "2050-01-01T00:00:00Z" },
+            "StartDate": { "DateTime": "2017-01-01T00:00:00Z" }
+          },
+          {
+            "ServiceTaskSchedules": {
+              "ServiceTaskSchedule": [
+                {
+                  "ScheduleDescription": null,
+                  "LastInstance": {
+                    "OriginalScheduledDate": { "DateTime": "2022-10-09T23:00:00Z" },
+                    "CurrentScheduledDate": { "DateTime": "2022-10-10T05:00:00Z" },
+                    "Ref": { "Value": { "anyType": [ "22281556", "8318" ] } }
+                  },
+                  "StartDate": { "DateTime": "2018-09-02T23:00:00Z" },
+                  "EndDate": { "DateTime": "2050-01-01T00:00:00Z" },
+                  "NextInstance": null
+                },
+                {
+                  "EndDate": { "DateTime": "2050-01-01T00:00:00Z" },
+                  "NextInstance": {
+                    "Ref": { "Value": { "anyType": [ "22295512", "8325" ] } },
+                    "OriginalScheduledDate": { "DateTime": "2022-10-16T23:00:00Z" },
+                    "CurrentScheduledDate": { "DateTime": "2022-10-16T23:00:00Z" }
+                  },
+                  "StartDate": { "DateTime": "2018-09-02T23:00:00Z" },
+                  "ScheduleDescription": "Monday every other week",
```

Comment:
Is that right?

Reply:
The code ignores the schedule level descriptions for Kingston/Sutton, yeah - CobrandSLWP.pm calls `_parse_schedules($task, 'task')` to only use the task-level description.